pkg/seedadmissioncontroller/webhooks/admission/extensionresources/templates_test.go

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package extensionresources_test
+
+var (
+	backupBucketFmt = `
+{ "apiVersion": "extensions.gardener.cloud/v1alpha1",
+"kind": "BackupBucket",
+"metadata": {
+  "name": "cloud--gcp--fg2d6",
+  %s
+  "namespace": "prjswebhooks"
+  },
+"spec":{
+  "type": %q,
+  "region": "eu-west-1",
+  "secretRef": {
+    "name": %q,
+    "namespace": "garden"
+  }
+}
+}
+`
+
+	backupEntryFmt = `
+{
+  "apiVersion": "extensions.gardener.cloud/v1alpha1",
+  "kind": "BackupEntry",
+  "metadata": {
+    "name": "shoot--foobar--gcp--sd34f",
+	%s
+    "namespace": "prjwebhooks"
```

Comment:
The extensions.gardener.cloud.BackupBucket resource is not a namespaced one.

```suggestion
```

Reply:
According to the `ValidateBackupEntryUpdate` method this is not true, it does check for _namespace_

Reply:
Can you share the code where this magic happens?

Reply:
`github.com/gardener/gardener/pkg/apis/core/validation/backupentry.go`:27 where the second argument apparently is `requiresNamespace` (line: `apivalidation.ValidateObjectMeta(&backupEntry.ObjectMeta, true, validateBackupEntryName, field.NewPath("metadata"))...)`)

Reply:
core.BackupEntry != extensions.BackupEntry, right? :)

Reply:
That is correct! But this is also correct for `github.com/gardener/gardener/pkg/apis/extensions/validation/backupentry.go`: 28

Reply:
Nice catch! It should be called with `requiresNamespace=false`. Can you amend this change to your PR?

Reply:
Yes, I can and will.

Reply:
> Should I do it for both core and extensions packages ?

Only for the extensions package, core.BackupEntry is namespaced.

```
NAME                               SHORTNAMES      APIVERSION                           NAMESPACED   KIND
backupentries                      be              extensions.gardener.cloud/v1alpha1   false        BackupEntry
```

```
NAME                               SHORTNAMES        APIVERSION                               NAMESPACED   KIND
backupentries                      bec               core.gardener.cloud/v1beta1              true         BackupEntry
```